packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -647,9 +630,15 @@ export class MgtFileList extends MgtTemplatedComponent {
         <fluent-progress-ring role="progressbar" viewBox="0 0 8 8" class="progress-ring"></fluent-progress-ring>
       `;
     } else {
-      return html`<a id="show-more" class="show-more" @click=${() => this.renderNextPage()} tabindex="0" @keydown=${
-        this.onShowMoreKeyDown
-      }><span>${this.strings.showMoreSubtitle}<span></a>`;
+      return html`
+        <a
```

Comment:
Nit pick, but this should be a button and not link as it's changing state not navigating.